<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoSearch</class>
 <widget class="QWidget" name="InfoSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Packet Names</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="lblMessage">
     <property name="styleSheet">
      <string notr="true">color: rgb(5, 17, 127);
font: italic 10pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Enter a comma separated list of mobile numbers in the textbox below or click on &quot;Import From File&quot; to import from a text file.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnGetFolder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(5, 17, 127);
text-decoration: underline;
font: 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Click Here To Select FEP/MNO Folder</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFolder">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 255);
font: italic 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblMobileNos">
       <property name="styleSheet">
        <string notr="true">color: rgb(5, 17, 127);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Mobile Numbers:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtMobileNos"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnGetFile">
       <property name="styleSheet">
        <string notr="true">color: rgb(5, 17, 127);
text-decoration: underline;
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Import From File</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="styleSheet">
        <string notr="true">color: rgb(5, 17, 127);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset resource="rsrc.qrc">
         <normaloff>:/img/find.png</normaloff>:/img/find.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 255);
font: italic 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Loading</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblResult">
       <property name="styleSheet">
        <string notr="true">color: rgb(5, 17, 127);
font: 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>List of Items Found:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tblResult">
       <property name="toolTip">
        <string>Click on a row to view subscriber information</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>140</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtMobileNos</tabstop>
  <tabstop>btnGetFile</tabstop>
  <tabstop>btnSearch</tabstop>
 </tabstops>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
